c++/samples/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+cmake_minimum_required (VERSION 2.8.12)
+
+project ("Cap'n'proto sample" CXX)
+
+# Needed to include generated Cap'n'proto headers
+set (CMAKE_INCLUDE_CURRENT_DIR ON)
+
+# Include CMake scripts
+include (FindPkgConfig)
+set (CMAKE_MODULE_PATH
+    "${PROJECT_SOURCE_DIR}/../cmake"
+    "${PROJECT_SOURCE_DIR}/cmake"
+)
+
+# Enable tests
+include(CTest)
+
+# Check for C++11 support
+include (C++11)
```

Comment:
I'm not sure this is useful. The macro doesn't work with MSVC, and support for "-std=c++11" doesn't necessarily guarantee *enough* C++11 support.

The checks at the top of `kj/common.h` do a more thorough job, and will halt the build with an error message fairly quick.